dashboard/package.json

```diff
@@ -15,6 +15,7 @@
     "@storybook/addon-postcss": "^2.0.0",
     "axios": "^0.24.0",
     "formik": "^2.2.9",
+    "moment": "^2.29.4",
```

Comment:
[Do we really need momentjs?](https://you-dont-need.github.io/You-Dont-Need-Momentjs/)